tasks/main.yml

```
@@ -18,7 +18,7 @@
     state: present
 
 - name: Check and prepare role variables
-  include_tasks: check-and-prepare-role-variables.yml
+  include_tasks: shell_{{ ha_cluster_pacemaker_shell }}/check-and-prepare-role-variables.yml
```

```suggestion
  include_tasks: shell_{{
    ha_cluster_pacemaker_shell }}/check-and-prepare-role-variables.yml
```
or
```
include_tasks: shell_{{ ha_cluster_pacemaker_shell }}/\
  check-and-prepare-role-variables.yml
```
or 
```
include_tasks: shell_{{ ha_cluster_pacemaker_shell }}/check-and-prepare-role-variables.yml  # noqa yamllint[line-length]
```
or see https://linux-system-roles.github.io/documentation/howto/working-with-ansible-jinja-code-and-filters.html